### ISSUE TITLE
[Bug Report] Vuetify v-btn component stays in active state after click

### ISSUE THREAD

**Issue by Stoom** [vuetifyjs/vuetify#11354]
### Environment
**Vuetify Version:** 2.2.27
**Vue Version:** 2.6.11
**Browsers:** Chrome 81.0.4044.113
**OS:** Windows 10

### Steps to reproduce
Create a button in an app bar and associate a router `to` link.  Once clicked the route is changed but since the button was outside of the router view port it stays active.

### Expected Behavior
The button functions as it does without a `to` link

### Actual Behavior
The button continues to have the `v-btn--active` class on it

### Reproduction Link
![image](https://user-images.githubusercontent.com/5356206/81411056-6a279900-9107-11ea-9251-3960380649ec.png)

This is also the same as #8435 but was automatically closed 😠

**Comment by jacekkarczmarczyk** [comment]
You've submitted an issue that does not contain a **valid** or **simple** reproduction. In order to effectively disposition this, we need to verify that the problem exists **outside** of your environment concisely as possible. This issue will be tagged as `needs reproduction` for the next 48 hours before being closed.

You can find information on how to contribute to Vuetify below:

- [Vuetify Contributing Guide](https://vuetifyjs.com/getting-started/contributing/)
- [Contributing Rules](https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md)
- [Codepen](https://template.vuetifyjs.com/)
- [Codepen w/Router](https://codepen.io/zikeji/pen/ypeQNm)
- A **simple** and **small** repository

Keep in mind we cannot debug a personal application. Thank you for your understanding. If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com/).

**Comment by Stoom** [comment]
I found the issue.  When it's a router link and the route is that path then the button is active (https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/mixins/routable/index.ts#L49).  It also looks like there's no way to deactivate this behavior.

Is there something I'm missing on how to deactivate the `v-btn--active` class when it's a router link?  I've tried both `active-class` and `exact-active-class` with no luck
